<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadFromMongoDbDialog</class>
 <widget class="QDialog" name="LoadFromMongoDbDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New column name</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="dbSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="dbNameLb">
       <property name="text">
        <string>database name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dbNameEdit">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dbIpLb">
       <property name="text">
        <string>ip: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dbIpEdit">
       <property name="text">
        <string>localhost</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="portLb">
       <property name="text">
        <string>port: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="dbPortSpinBox">
       <property name="maximum">
        <number>65536</number>
       </property>
       <property name="value">
        <number>50051</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBoxLayout">
     <item>
      <spacer name="buttonBoxSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
